src/uvwasi.c

```diff
@@ -2325,7 +2325,7 @@ uvwasi_errno_t uvwasi_poll_oneoff(uvwasi_t* uvwasi,
                                   uvwasi_size_t* nevents) {
   struct uvwasi_poll_oneoff_state_t state;
   struct uvwasi__poll_fdevent_t* fdevent;
-  uvwasi_userdata_t timer_userdata;
+  uvwasi_userdata_t timer_userdata = 0;
```

Comment:
Very small nit: would you mind separating the variable declaration from the initialization.

Reply:
I've separated it now. I placed the initialization after the other declarations, hope that is the right thing to do.